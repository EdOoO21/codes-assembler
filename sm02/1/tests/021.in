-144 412 -144 300 -424 -19 727 -558 623 -28 654 972 49 -625 34 80 -517 614 -994 -972 -241
