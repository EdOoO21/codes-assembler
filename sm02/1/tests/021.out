-241
-972
-994
614
-517
80
34
-625
49
972
654
-28
623
-558
727
-19
-424
300
-144
412
-144
